### PR TITLE
Add check-mysql-threads.rb script

### PR DESCRIPTION
**Pull request by tanji**
Added simple script to check running thread count.

### CODE REVIEW

File: bin/check-mysql-threads.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# MySQL Health Plugin
+# ===
+#
+# This plugin evaluates the number of MySQL running threads and warns you according to specified limits
+#
+# Author: Guillaume Lefranc <guillaume@mariadb.com>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
```
Comment (eheydrick) [inline bin/check-mysql-threads.rb]: can you use the standard header from http://sensu-plugins.io/docs/developer_guidelines.html.

**Comment by eheydrick** [comment]
Thanks for the new plugin! Can you get the rubocop tests passing so we can merge.

**Comment by tanji** [comment]
@eheydrick - tests are passing now. Thanks!

**Comment by eheydrick** [comment]
Awesome, thanks @tanji!

**Pull request merged**